<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParcelInfoStatisticDialog</class>
 <widget class="QDialog" name="ParcelInfoStatisticDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>12</y>
     <width>781</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>510</y>
     <width>781</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="close_button">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>490</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="print_button">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>490</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>print</string>
   </property>
  </widget>
  <widget class="QComboBox" name="organization_type_cbox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Organization Type</string>
   </property>
  </widget>
  <widget class="QComboBox" name="organization_cbox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>50</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Organization</string>
   </property>
  </widget>
  <widget class="QComboBox" name="department_cbox">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>50</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>30</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Department</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>191</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>sort by duureg</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>180</y>
     <width>191</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>sort by department</string>
   </property>
  </widget>
  <widget class="QPushButton" name="find_button">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>209</y>
     <width>771</width>
     <height>281</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="ubgis_stat_tab">
    <attribute name="title">
     <string>UBGIS Status change</string>
    </attribute>
    <widget class="QTableWidget" name="result_twidget">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>3</y>
       <width>760</width>
       <height>250</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Duureg</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Department</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Parcel Count</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Not Fixed</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>In process</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Finished</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Done</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="ubgis_finish_tab">
    <attribute name="title">
     <string>UBGIS Finish user</string>
    </attribute>
    <widget class="QTableWidget" name="result_user_twidget">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>3</y>
       <width>760</width>
       <height>250</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>User name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Lastname</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Firstname</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Count</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
  <widget class="QComboBox" name="user_name_cbox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>291</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>User name</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="finish_date">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="finish_date_checkbox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>131</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Is Finish Date</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
